lib/ocpp/v201/utils.cpp

```diff
@@ -183,6 +183,27 @@ bool is_critical(const std::string& security_event) {
     return false;
 }
 
+std::set<ChargingProfilePurposeEnum> get_purposes_to_ignore(const std::string& csl, const bool is_offline) {
+    if (not is_offline) {
+        return {};
+    }
+
+    if (csl.empty()) {
+        return {};
+    }
```

Comment:
could be simplified:
```
if (!is_offline() || csl.empty()) {
  return {};
}
```
Some coding standards recommend avoiding having multiple return statements (MISRA goes further allowing a single return statement).

MISRA is generally looking for
```
std::set<ChargingProfilePurposeEnum> result{};
if (is_offline && !csl.empty()) {
    ... main for loop ...
}
return result;
```